editor/provider/index.js

```diff
@@ -0,0 +1,142 @@
+/**
+ * External dependencies
+ */
+import { bindActionCreators } from 'redux';
+import { Provider as ReduxProvider } from 'react-redux';
+import { Provider as SlotFillProvider } from 'react-slot-fill';
+import { flow, pick } from 'lodash';
+
+/**
+ * WordPress Dependencies
+ */
+import { createElement, Component } from '@wordpress/element';
+import { EditableProvider } from '@wordpress/blocks';
+import { APIProvider, PopoverProvider, DropZoneProvider } from '@wordpress/components';
+
+/**
+ * Internal Dependencies
+ */
+import { setupEditor, undo } from '../actions';
+import EditorSettingsProvider from '../settings/provider';
```

Comment:
Maybe the `EditorSettingsProvider` component is obsolete with this one and could be consolidated into it?

Reply:
eagle eye :) Agree with this, I'll update accordingly.